app/components/power-select/component.js

```diff
@@ -1,19 +1,19 @@
+import { action } from "@ember/object";
 import { isBlank } from "@ember/utils";
 import Ember from "ember";
 import PowerSelectComponent from "ember-power-select/components/power-select";
 
-/* istanbul ignore next */
-const PowerSelectCustomComponent = PowerSelectComponent.extend({
-  init(...args) {
-    this._super(...args);
+export default class PowerSelectCustomComponent extends PowerSelectComponent {
+  constructor(...args) {
+    super(...args);
 
-    this.set("extra", this.getWithDefault("extra", {}));
-    this.set("extra.testing", Ember.testing);
-  },
+    this.extra = this.args.extra ?? {};
+    this.extra.testing = Ember.testing;
```

Comment:
Not sure if `Ember.testing` still exsist

Reply:
Yes it still does, but it should get replace by `@embroider/macroCondition` in the future. I will note this and file it in on of the upcoming PRs.